### PR TITLE
[GH]Add initial PR template.

### PR DESCRIPTION
**Pull request by juanfra**
This is experimental. I'm adding a PR template with what I consider it's the most common things we need in PRs, given our workflow. Also, trying not to super bloat it, keeping it short so we have something consistent that devs can effectively check every time they open a PR.

Open to suggestions.

_Let's discuss here and then we replicate in the other repos_.

### CODE REVIEW

**Comment by Camwyn** [comment]
Personally, I'd like artifacts after most of the text. About where "etc" line is. That way a reviewer reads what to look for before they look at the images?

**Comment by juanfra** [comment]
@Camwyn nice! Makes sense, pushed the changes 🙌

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+<!-- Bug fix (non-breaking change which fixes an issue) -->
+<!-- New feature (non-breaking change which adds functionality) -->
+<!-- Include any important information for reviewers -->
+<!-- Etc, etc, etc -->
+
+### Artifacts <!-- if applicable-->
+<!-- 🎥 screencast or 📷 screenshots -->
+
+### Checklist:
+- [ ] I've included a readme entry. <!-- Confirm that it includes the ticket ID -->
```
Comment (sc0ttkclark) [inline .github/PULL_REQUEST_TEMPLATE.md:16]:
```suggestion
- [ ] I've included a readme.txt Changelog entry. <!-- Confirm that it includes the ticket ID -->
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+### Ticket
+🎫 [TICKET_ID] <!-- Ticket ID, if there's any put it between brackets -->
+
+### Description
+<!-- Please describe what you have changed or added -->
+<!-- What types of changes does your code introduce? -->
+<!-- Bug fix (non-breaking change which fixes an issue) -->
+<!-- New feature (non-breaking change which adds functionality) -->
+<!-- Include any important information for reviewers -->
+<!-- Etc, etc, etc -->
```
Comment (sc0ttkclark) [inline .github/PULL_REQUEST_TEMPLATE.md:10]: Maybe we should create a "good" PR and link to it here so people have an example?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+### Artifacts <!-- if applicable-->
+<!-- 🎥 screencast or 📷 screenshots -->
```
Comment (sc0ttkclark) [inline .github/PULL_REQUEST_TEMPLATE.md:13]:
```suggestion
<!-- 🎥 screencast(s) or 📷 screenshot(s) -->
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+### Checklist:
+- [ ] I've included a readme entry. <!-- Confirm that it includes the ticket ID -->
+- [ ] My code is tested. <!-- Check that tests are passing and DO NOT merge if they're failing. -->
+- [ ] My code has proper inline documentation.
```
Comment (sc0ttkclark) [inline .github/PULL_REQUEST_TEMPLATE.md:18]: We could link this 'proper inline documentation' text to the TEC coding standards page for that

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
+### Ticket
+🎫 [TICKET_ID] <!-- Ticket ID, if there's any put it between brackets -->
```
Comment (sc0ttkclark) [inline .github/PULL_REQUEST_TEMPLATE.md:2]:
```suggestion
<!-- Ticket ID, if there's any put it between brackets -->
🎫 [TICKET_ID]
```

**Review by sc0ttkclark — COMMENTED** [review]
Definitely looks good!

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (juanfra) [reply .github/PULL_REQUEST_TEMPLATE.md:12]: That's a neat idea, maybe we can add this once we have that PR?

**Review by sc0ttkclark — APPROVED** [review]

**Pull request merged**